src/workflows-app/RunDetails.js

```diff
@@ -151,6 +157,30 @@ export const BaseRunDetails = (
     [signal, workspaceId]
   );
 
+  const renderInProgressElement = (workflow) => {
+    if (workflow.status === 'Running') {
+      return span({ style: { fontStyle: 'italic' } }, ['In progress - ']);
+    }
+  };
+
+  const calculateTotalCost = (callObjects) => {
+    let total = 0;
+    Object.values(callObjects).forEach((call) => {
+      if (!call[0].taskStartTime) {
```

Comment:
Why is there an index inside a call? My guess is that this Is this because of shards of a scatter and automated retries

So... I assume we want to sum all of the shards and retries, not just take the first one?

Reply:
I don't think this is a subworkflows thing, this is a "plain old scattered task" thing. I worry that not only are we "not including subworkflows", we are also "ignoring every task in scatters except the first index".

Luckily, I think you can easily fix this by just iterating over all indexes in the `call` array, like eg
```js
 Object.values(callObjects).forEach((call) => {
    call.forEach((shard) => {
      if (!shard.taskStartTime) {
      [...]
```